### DISCUSSION TITLE
Add debug source code tutorial

### DISCUSSION THREAD

**Discussion by T-TRz879** [juicedata/juicefs#2623] [Q&A]

**Comment by zhijian-pro** [comment]
Just debug it like a regular go program.
Note that juicefs debugged in the IDE must be  without the `background` parameter, so you cannot debug `juicefs mount -d`.
For example, debugging in Goland:

<img width="1527" alt="image" src="https://user-images.githubusercontent.com/31313340/187135135-bc0b91bf-b280-42d2-acd0-ac2d40b99bff.png">

**Reply by T-TRz879** [reply]
this issue is aim to commit my pr
